src/CaptureClient/CMakeLists.txt

```diff
@@ -41,6 +43,7 @@ target_compile_options(CaptureClientTests PRIVATE ${STRICT_COMPILE_FLAGS})
 
 target_sources(CaptureClientTests PRIVATE
         ApiEventProcessorTest.cpp
+        CaptureSaveToFileProcessorTest.cpp
```

Comment:
Rename to `SaveToFileCaptureEventProcessorTest.cpp`.

Reply:
done.